frontend/src/bundles/common/components/badge/components/badge-icon/badge-icon.tsx

```diff
@@ -2,14 +2,21 @@ import { HeadphonesOutlined } from '@mui/icons-material';
 import { SvgIcon } from '@mui/material';
 import { type ReactElement } from 'react';
 
+import styles from '../../styles.module.scss';
+
 type Properties = {
     icon?: ReactElement;
     iconClass?: string;
+    HRbadge?: boolean;
```

Comment:
Name boolean variable like is..., has... etc

Reply:
done